bridge/slack/handlers.go

```diff
@@ -284,11 +289,10 @@ func (b *Bslack) handleDownloadFile(rmsg *config.Message, file *slack.File) erro
 	return nil
 }
 
-func (b *Bslack) fileIsAvailable(file *slack.File) bool {
-	// Only download a file if it is not in the cache or if it has been entered more than a minute ago.
-	if ts, ok := b.cache.Get("file" + file.ID); ok && time.Since(ts.(time.Time)) > time.Minute {
+func (b *Bslack) fileIsRecentlyCached(file *slack.File) bool {
```

Comment:
Nit: to make this properly named it should be `fileWasRecentlyCached()`.

Reply:
haha depends if one reads with an implied hyphen or space, but fair enough :)

Reply:
Actually, I'm unhappy with my stated rationale, but this captures an explanation for my underlying sensibilities :) 

https://stackoverflow.com/questions/3669433/do-you-use-tense-when-naming-methods-of-boolean-return-type

Changed already , but just food for thought